src/recovery/tally.rs

```diff
@@ -476,9 +479,25 @@ pub fn recover_ledger_from_logs(
                 None
             }
         };
-        if let Some(new_fragment) = new_fragment {
-            ledger = ledger.apply_fragment(&ledger.get_ledger_parameters(), &new_fragment, block_date)
-                .expect("Should be impossible to fail, since we should be using proper spending counters and signatures");
+        if let Some((new_fragment, wlt)) = new_fragment {
+            let new_ledger =
+                ledger.apply_fragment(&ledger.get_ledger_parameters(), &new_fragment, block_date);
+            match new_ledger {
+                Ok(new_ledger) => {
+                    if let Some(wlt) = wlt {
```

Comment:
What is `wlt`?

Reply:
Short for `wallet`, let me rename it so it's easier to read. I used it in some places to avoid wrapping the line, but here is not necessary

Reply:
> to avoid wrapping the line

Plug in rustfmt as the autoformatter already :)

Reply:
I do, I just like to avoid having multiple lines with parenthesis that feels in the wrong place like:
```
// rustfmt produced smth like this
long line if
{
  stuff
}
```
if this can be avoided by a few characters